packages/api/src/hooks/useAccountsList.ts

```diff
@@ -30,11 +32,20 @@ const useAccountsList = () => {
                 is_wallet: account.account_category === 'wallet',
                 /** The account ID of specified account. */
                 loginid: `${account.loginid}`,
+                /** This represents the current status of the proof of address document submitted for authentication. */
+                poa_status: get_account_status_data?.authentication?.document?.status,
+                /** This represent the current status for proof of identity document submitted for authentication. */
+                poi_status: get_account_status_data?.authentication?.identity?.status,
                 /** Account's currency config information */
                 currency_config: account.currency ? getConfig(account.currency) : undefined,
             } as const;
         });
-    }, [authorize_data.account_list, authorize_data.loginid]);
+    }, [
+        authorize_data.account_list,
+        authorize_data.loginid,
+        get_account_status_data?.authentication?.document?.status,
+        get_account_status_data?.authentication?.identity?.status,
```

Comment:
@adrienne-deriv Add a separate `useMemo` to add these data, Similar to what we are doing with `modified_accounts_with_balance`.